x-pack/plugins/security_solution/cypress/screens/timeline.ts

```diff
@@ -24,6 +24,10 @@ export const CASE = (id: string) => {
   return `[data-test-subj="cases-table-row-${id}"]`;
 };
 
+export const SELECT_CASE = (id: string) => {
```

Comment:
Could you please delete the

```
export const CASE = (id: string) => {
  return `[data-test-subj="cases-table-row-${id}"]`;
};
```

function in the same file? With your changes It is not used anymore.